protocol/message_validator.go

```diff
@@ -2,11 +2,126 @@ package protocol
 
 import (
 	"errors"
+	"strconv"
 	"strings"
 
 	"github.com/status-im/status-go/protocol/protobuf"
 )
 
+func ValidateReceivedPairInstallation(message *protobuf.PairInstallation) error {
+	if message.Clock == 0 {
+		return errors.New("Clock can't be 0")
```

Comment:
Errors should be lower case so that later they are joined in this manner: `error1 due to reason 1: error2 due to reason 2`.

```suggestion
		return errors.New("clock can't be 0")
```